### ISSUE TITLE
nvda does not announce display of gmail settings page

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#15255]
### Steps to reproduce:
open the gmail website
gmail.com
enter any account
press ctrl + home
walk with the arrows below to the button:
settings
press enter

### Actual behavior:
nvda does not announce anything
the focus is also not moved

### Expected behavior:
From what I remember of the previous behavior, nvda moved the focus to the new open page
I would like to opt this behavior again
The focus must be directed to the newly opened page.
As a suggestion, I would also like an nvda announcement, something like new content displayed.

to better explain the problem, as nothing happens, the user thinks that the button was not activated, or that pressing the button did not produce even an action.
This is compounded as the new content appears at the bottom of the page.

#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-28841,a85afcef

#### Windows version:
10 19045.3271
#### Name and version of other software in use when reproducing the issue:
chrome.exe, Google Chrome 115.0.5790.171

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
[gmail.txt](https://github.com/nvaccess/nvda/files/12252183/gmail.txt)

**Comment by seanbudd** [comment]
Closing as duplicate of #14889
